src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -164,6 +164,75 @@
         {
             "name": "stepfunctions_previewstatemachine",
             "description": ""
+        },
```

Comment:
Our telemetry has two override files??
I'm deleting the one in the common repo.